files/rsyslog/60-mmk8s.conf

```diff
@@ -17,31 +17,31 @@ if ((strlen($!CONTAINER_NAME) > 0) and (strlen($!CONTAINER_ID_FULL) > 0)) or
             action(type="mmnormalize" rulebase="/etc/rsyslog.d/multiline-json.rulebase")
             foreach ($.ii in $!multilinejson) do {
                 if strlen($!@timestamp) == 0 then {
-                    set $!@timestamp = $.ii!time;
+                    set $.all!@timestamp = $.ii!time;
                 }
                 if strlen($!stream) == 0 then {
-                    set $!stream = $.ii!stream;
+                    set $.all!stream = $.ii!stream;
                 }
                 if strlen($!log) == 0 then {
-                    set $!log = $.ii!log;
+                    set $.all!log = $.ii!log;
                 } else {
-                    reset $!log = $!log & $.ii!log;
+                    reset $.all!log = $.log & $.ii!log;
```

Comment:
`reset $.all!log = $.all!log & $.ii!log;`
If it is a problem that `$.all!log` does not exist before calling reset on it, then maybe something like this:
`} else if strlen($.all!log) == 0 then {
  set $.all!log = $.ii!log;
} else {
  reset $.all!log =  $.all!log & $.ii!log;
}
```

Reply:
Thank you, @richm.  Yes, it is a bug...  Fixed it.